stripe-react-native.podspec

```diff
@@ -1,7 +1,7 @@
 require 'json'
 
 package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
-stripe_version = '23.6.0'
+stripe_version = '~> 23.7.0'
```

Comment:
any reason why we don't use `23.7.0` ?

Reply:
this matches up to but not including 23.8.0, so we'll get in any patch fixes that `stripe-ios` releases. it's the equivalent of the android's `20.24.+`

Reply:
ok, will apply the similar change in identity